SConstruct

```diff
@@ -378,6 +380,34 @@ elif env['COMPILER'] == 'msvc':
         env.Append(CCFLAGS=Split('/Od /Zi /MD'))
         env.Append(LINKFLAGS=Split('/DEBUG'))
 
+if not env['SHOW_CMDS']:
+    # Hide long compile lines from the user
+    arch = env['MACOS_ARCH'] if IS_DARWIN else 'x86_64'
+    env['CCCOMSTR']     = 'Compiling {} $SOURCE ...'.format(arch)
+    env['SHCCCOMSTR']   = 'Compiling {} $SOURCE ...'.format(arch)
+    env['CXXCOMSTR']    = 'Compiling {} $SOURCE ...'.format(arch)
+    env['SHCXXCOMSTR']  = 'Compiling {} $SOURCE ...'.format(arch)
+    env['LINKCOMSTR']   = 'Linking {} $TARGET ...'.format(arch)
+    env['SHLINKCOMSTR'] = 'Linking {} $TARGET ...'.format(arch)
+    env['LEXCOMSTR']    = 'Generating $TARGET ...'
+    env['YACCCOMSTR']   = 'Generating $TARGET ...'
+    env['RCCOMSTR']     = 'Generating $TARGET ...'
+    if env['COLOR_CMDS']:
+        ansi_bold = '\033[1m'
+        ansi_bold_green     = '\033[32;1m' + ansi_bold
+        ansi_bold_red       = '\033[31;1m' + ansi_bold
+        ansi_bold_yellow    = '\033[33;1m' + ansi_bold
+
+        env['CCCOMSTR']     = ansi_bold_green + env['CCCOMSTR']
```

Comment:
I can't see how the color is reset ? I was expecting something like the following:
```
env['CCCOMSTR'] = ansi_bold_green + env['CCCOMSTR'] + ansi_normal
``` 
otherwise the color remains for the following text, may be this is expected ?

Reply:
Good catch! I'll fix that